content/js1/sprints/1/prep/index.md

```diff
@@ -8,4 +8,443 @@ backlog= 'Module-JS1'
 backlog_filter= 'Week 1'
 +++
 
+## Prerequisites
 
+- Basic arithmetic
+- Access to a Linux/MacOS terminal
+
+## Learning objectives
+
+- Define a terminal
+- Describe the purpose of VSCode
```

Comment:
We don't use or talk about VSCode at all in this text.

Reply:
VSCode is mentioned in the Git resources at the start of the course, so I don't think we need this learning objective here